assets/sass/cds/_sections.scss

```diff
@@ -424,6 +424,15 @@ span.livefr {
     }
   }
 
+  h3.live::after,
```

Comment:
it looks like on the tools and resources page there's a style that sets the font colour to black.

`.work-in-progress--product-module`
Looks like it will need an addition for h3 as well (it has h1, h2, and h4 versions)

Reply:
done